doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/ParserConfiguration.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.maven.doxia.siterenderer;
+
+import java.util.function.Consumer;
+
+import org.apache.maven.doxia.parser.Parser;
+
+/**
+ * Configures a Doxia {@link Parser}.
+ * @since 2.0.0
+ */
+@FunctionalInterface
+public interface ParserConfiguration extends Consumer<Parser> {
+
+    /**
+     * Applies the configuration to the given parser.
+     * @param parser the parser to configure
+     */
+    @Override
+    void accept(Parser parser);
```

Comment:
Wouldn't `configure()` be more reasonable here?

Reply:
Right, I was thinking about implementing a default Java interface (Consumer) but method name `accept` is really weird in this context and converting between different functional interfaces is easy enough with lambdas.